controllers/post.controller.js

```diff
@@ -17,13 +44,54 @@ export const createPost = async (req, res) => {
     }
 };
 
+export const getPosts = async (req, res) => {
+    try {
```

Comment:
We need new version to find the posts .. i mean each way for annoucements and feeds